src/entities/index.ts

```diff
@@ -50,6 +50,14 @@ const setup = (
     },
     { inline: true },
   );
+  Then(
+    'the documents for the {entity} with (\\{.*\\}) {op}',
+    async (entity, query, op, payload) => {
+      const docs = await entities[entity].find(JSON.parse(query));
+      compare(op, docs, payload);
```

Comment:
my only concern here is: does `compare` support multiple documents within all the ops? AFAIK you are just testing with `includes` which in fact looking at the implementation supports multiple docs, but there are some that might not support it (such as `has keys` for instance)

Reply:
there are no changes in the compare func, it supports _includes_ and _is_ as usual. The only difference is that the value to compare is an array of documents instead of just one document.

Reply:
ok good to know! My point was actually more focused on the tests themselves. I think that your are just using the `includes` `{op}` within all your tests of the new step `the documents for the {entity} with (\\{.*\\}) {op}`. What I was saying, with the way the step is defined, is that if you actually try it with another `{op}` such as `has keys` *I believe* it won't work as expected. I haven't tested it but it seems that there are some `{op}`s that unfortunately don't consider that the two operands can be arrays

Reply:
yeah, we should add new ops specifically for arrays. 
Or maybe add new steps like:
```
'some of the documents for the {entity} with (\\{.*\\}) {op}',
'all of the documents for the {entity} with (\\{.*\\}) {op}',
```
so we can match any or all of the docs